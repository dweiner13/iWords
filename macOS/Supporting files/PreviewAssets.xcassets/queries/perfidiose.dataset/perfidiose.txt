perfidios.e          ADJ    1 1 VOC S M POS             
perfidiosus, perfidiosa, perfidiosum  ADJ    lesser
treacherous;
e                    SUFFIX                             
-ly; -ily;  Converting ADJ to ADV
perfidios.e          ADV    POS                         
perfidiosus, perfidiosa, perfidiosum  ADJ    lesser
treacherous;

