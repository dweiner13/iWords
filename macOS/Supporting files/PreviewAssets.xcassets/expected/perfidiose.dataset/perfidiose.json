[{"definition":{"_0":{"words":[{"meaning":"treacherous;","expansion":{"adj":{"_0":"perfidiosus, perfidiosa, perfidiosum","_1":["lesser"]}}}],"truncated":false,"possibilities":[{"adjective":{"_0":{"number":"S","gender":"M","variety":1,"degree":"POS","text":"perfidios.e","declension":1,"case":"VOC"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"-ly; -ily;  Converting ADJ to ADV"}],"truncated":false,"possibilities":[{"suffix":{"_0":"e"}}]}}},{"definition":{"_0":{"words":[{"meaning":"treacherous;","expansion":{"adj":{"_0":"perfidiosus, perfidiosa, perfidiosum","_1":["lesser"]}}}],"truncated":false,"possibilities":[{"adverb":{"_0":{"degree":"POS","text":"perfidios.e"}}}]}}}]